monkey/agent_plugins/payloads/cryptojacker/src/bitcoin_mining_network_traffic_simulator.py

```diff
@@ -1,7 +1,21 @@
+import base64
+import json
+import logging
+import random
+import urllib.request
 from typing import Optional
 
+import urllib3
+
 from common.event_queue import IAgentEventPublisher
 from common.types import AgentID, SocketAddress
+from common.utils.code_utils import PeriodicCaller
+from infection_monkey.island_api_client.http_client import handle_island_errors
+
+REQUEST_INTERVAL = 10  # seconds
```

Comment:
Have we done any research on what a reasonable value for this is?

Reply:
Nope, this is a random value